source/sync/get-started.txt

```diff
@@ -92,31 +96,30 @@ type:
   object models and code that you use in your production app. This is useful if
   you prefer a client-first approach that uses idiomatic object models in your
   preferred programming language.
-  
-  For a walkthrough of how to turn on Development Mode, see :ref:`Enable
-  Development Mode <development-mode>`.
+
+  To get started, check out :ref:`create-sync-schema`.
 
 - **Generate object models from a document schema:** App Services can automatically
```

Comment:
I'm not sure "App Services" is a hard requirement here, either. I _assume_ that if I enable Device Sync from Data Services, I can still generate models without going through App Services, but I'm not sure - I haven't tried it.

Maybe we should generally scrub these two pages from "App Services" mentions - the more I look at this content, the more I think we need to genericize it to reflect the multiple entry points to Device Sync.